theia-extensions/viewer-prototype/src/browser/trace-server-preference.ts

```diff
@@ -1,8 +1,8 @@
 import { PreferenceSchema, PreferenceProxy, PreferenceScope } from '@theia/core/lib/browser';
 import { TRACE_SERVER_DEFAULT_PORT } from '../common/trace-server-url-provider';
 
-export const TRACE_PATH = 'trace-viewer.server.path';
-export const TRACE_PORT = 'trace-viewer.server.port';
+export const TRACE_PATH = 'trace Viewer.trace Server.path';
+export const TRACE_PORT = 'trace Viewer.trace Server.port';
```

Comment:
Thank you for the reply! Since you mentioned one change per commit above, is it better for me to have three commits?
The first commit: Add the "Server" label to the settings 
The second commit: Rename "Server" to "trace Trace Server" 
The third commit: Rename "trace-viewer" to "trace viewer"

Reply:
I don't think 3 commits is best since:
* Add the "Server" label to the settings, and
* Rename "Server" to "Trace Server"

can be simplified to:
* Add the "Trace Server" label to the settings

You don't need to keep traces of the PR revisions in the final merged commits.

Reply:
No need to overthink this... the PR with 2 commits look good now.